### PR TITLE
Minor cleanup of conditional components

### PR DESCRIPTION
**Pull request by eshmoylova**
Some small things noticed while reviewing #3556. Not sure if these are correct changes, but starting it as a PR is simpler than starting an issue and then a PR.

### CODE REVIEW

File: chapters/classes.tex [hunk 1]
```diff
@@ -408,15 +408,15 @@ \subsection{Conditional Component Declaration}\label{conditional-component-decla
 and the condition attribute is kept from the original declaration (see
 \cref{interface-compatibility-or-subtyping}).
 
-If the \lstinline!Boolean! expression is false the component (including its modifier) is removed from the flattened DAE, and connections to/from the component are removed.
+If the \lstinline!Boolean! expression is \lstinline!false!, the component (including its modifier) is removed from the flattened DAE, and connections to/from the component are removed.
```
Comment (eshmoylova) [inline chapters/classes.tex:411]: I think we usually highlight true and false in text.

File: chapters/classes.tex [hunk 2]
```diff
@@ -408,15 +408,15 @@ \subsection{Conditional Component Declaration}\label{conditional-component-decla
...
 A component declared with a condition-attribute can only be modified and/or used in connections.
 
 \begin{nonnormative}
 Adding the component and then removing it ensures that the component is valid.
 
 If a \lstinline!connect!-equation defines the connection of a non-conditional component \lstinline!c1! with a conditional component \lstinline!c2! and \lstinline!c2! is de-activated, then \lstinline!c1! must still be a declared element.
 
-There are annotations to handle the case where the connector should be connected when activated, see \cref{annotations-for-the-graphical-user-interface}.
+There are annotations to handle the case where the connector should be connected when activated, see \cref{connection-restrictions}.
```
Comment (eshmoylova) [inline chapters/classes.tex:419]: I could not find any annotations in graphical user interface that would apply to this case. I think it should be conneciton-restrictions.

Comment (HansOlsson) [reply chapters/classes.tex:419]: Yes, I must have missed that they were in that section.

**Review by HansOlsson — APPROVED** [review]
Good.

**Comment by eshmoylova** [comment]
@HansOlsson it shows for me that I am not authorized to merge this pull request:
image
Did I select the wrong branch to merge into?

**Comment by HansOlsson** [comment]
> @HansOlsson it shows for me that I am not authorized to merge this pull request: image Did I select the wrong branch to merge into?

I don't see that you did anything wrong.

I would guess it is one of the following:
- You are not member of some group; which seems odd given that you have review permissions etc. But I will check.
- GitHub is pushing more for 2FA - it could be something related to that.

**Comment by HansOlsson** [comment]
> I would guess it is one of the following:
> 
> * You are not member of some group; which seems odd given that you have review permissions etc. But I will check.
> * GitHub is pushing more for 2FA - it could be something related to that.

It was actually the first; currently you have role "Write" - but the master branch is protected and needs "Maintainer" role. Since we haven't detected it earlier I think we can leave it as; and will just have to be more alert in merging.

**Pull request merged**